Source/GRChomboCore/BoundaryConditions.hpp

```diff
@@ -72,8 +72,9 @@ class BoundaryConditions
         std::array<int, NUM_DIAGNOSTIC_VARS>
             vars_parity_diagnostic; /* needed only in AMRInterpolator */
         std::array<double, NUM_VARS> vars_asymptotic_values;
-        std::vector<int> mixed_bc_extrapolating_vars;
-        std::vector<int> mixed_bc_sommerfeld_vars;
+        std::map<int, int> mixed_bc_vars_map;
+        // std::vector<int> mixed_bc_extrapolating_vars;
+        // std::vector<int> mixed_bc_sommerfeld_vars;
```

Comment:
remove these if not used?

Reply:
done (and also `BoundaryConditions::m_diagnostic_comps`)